test/performance/README.md

```diff
@@ -1,128 +1,189 @@
-## Stress Test Experiments
+## Workload Benchmark for KubeStellar
 
+*Pre-requisite*: in order to follow the instructions below, you must have an environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
 
-## Generate the workload traffic
+To generate the sample workload for KubeStellar performance experiments proceed as following:
 
-1. Clone the following fork of the clusterloader2 repo: 
+### a) Generate the workload traffic
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+1. Clone the following clusterloader2 repo: 
 
-2. Configure the parameters of your experiment:  
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `WORK_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) If using plain K8s clusters:
+
+   ```bash
+   $ ./setup-clusterloader2.sh
+   ```
+
+   In plain Kubernetes environments, a modified version of the kube-burner [cluster-density](https://github.com/kube-burner/kube-burner/tree/main/examples/workloads/cluster-density) workload will be generated per namespace. This workload consists of the following objects:
```

Comment:
"will be" begs the question of agency. _what_ will do that, _when_? When these are not explicitly mentioned then the natural implication for the reader to draw is that the described action will take place as part of the command just described (invoking the setup script).

Reply:
ok, I see! Then, to avoid any ambiguity, I moved that content to `step-4` in the instructions.